File: output/gpt_4o/codenet/direct_translation/java/python/s301480578.py

n = int(input())
h = list(map(int, input().split()))
max_val = 0
l = 0
while l < n - 1:
    count = 0
    while l < n - 1 and h[l] >= h[l + 1]:
        l += 1
        count += 1
    max_val = max(max_val, count)
    l += 1
print(max_val)

File: dataset/codenet/Java/Code/s301480578.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        @SuppressWarnings("resource")
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] h = new int[n];
        for (int i = 0 ; i < n ; i++) {
            h[i] = sc.nextInt();
        }
        int max = 0;
        for (int l = 0 ; l < n - 1 ; l++) {
            int count = 0;
            while (l < n - 1 && h[l] >= h[l + 1]) {
                l++;
                count++;
            }
            max = Math.max(max, count);
        }
        System.out.println(max);
    }


}